### ISSUE TITLE
license change

### ISSUE THREAD

**Issue by Licenser** [dalibor-matura/safe-graph#1]
I'm not sure re-licensing someone else's code under a different license is in the spirit of open source and while I'm not a lawyer I don't think it conforms with either of the original licenses MIT/APL2.

**Comment by schulzch** [comment]
Yes, thats just illegal and GPL-ing a (static) library is pretty toxic to the ecosystem.
